Options.py

```diff
@@ -951,13 +951,18 @@ def __contains__(self, item):
 class OptionSet(Option[typing.Set[str]], VerifyKeys):
     default = frozenset()
     supports_weighting = False
+    random_str: str | None
 
-    def __init__(self, value: typing.Iterable[str]):
+    def __init__(self, value: typing.Iterable[str], random_str: str | None = None):
         self.value = set(deepcopy(value))
+        self.random_str = random_str
         super(OptionSet, self).__init__()
 
     @classmethod
     def from_text(cls, text: str):
+        check_text = text.lower().split(",")
+        if cls.valid_keys and len(check_text) == 1 and check_text[0].startswith("random"):
+            return cls({}, check_text[0])
```

Comment:
```suggestion
            return cls((), check_text[0])
```
I don't know why we would want a `dict` as the initializing iterable value.

Any of `{}` `[]` `set()` create a new object.
But `()` is a singleton, so it doesn't create a new object.